dll/win32/comdlg32/cdlg.h

```diff
@@ -199,8 +199,13 @@ HRESULT FileSaveDialog_Constructor(IUnknown *pUnkOuter, REFIID riid, void **ppv)
 
 /* Shared helper functions */
 void COMDLG32_GetCanonicalPath(PCIDLIST_ABSOLUTE pidlAbsCurrent, LPWSTR lpstrFile, LPWSTR lpstrPathAndFile) DECLSPEC_HIDDEN;
+#ifdef __REACTOS__
+int FILEDLG95_ValidatePathAction(void *fodInfos0, LPWSTR lpstrPathAndFile, IShellFolder **ppsf,
```

Comment:
Why does this parameter need to be void* ? is it because the underlying type is not available in the rest of comdlg32?

Reply:
`FileOpenDlgInfos` structure is defined in `filedlgbrowser.h` and it is an imperfect type in `cdlg.h`.
Forward declaration doesn't help.

Reply:
Improved.